src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -181,6 +183,16 @@ void MainWindow::open_project(const QString& filepath)
     m_project_manager.load_project(filepath.toAscii().constData());
 }
 
+void MainWindow::new_project()
+{
+    if (!can_close_project())
```

Comment:
agree
but maybe we don't want to close project if it's unsaved? Maybe it would be a good idea to introduce force flag here? And if force==false then throw exception